pkgs/development/python-modules/uni-curses/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, ncurses, x256, setuptools }:
+
+buildPythonPackage rec {
+  pname = "unicurses";
+  version = "2.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "Uni-Curses";
+    sha256 = "sha256-Anz5YMZULdMhZ802T3tj9LOcBZZfUML1fPOJbC47nVA=";
+  };
+
+  buildInputs = [ ncurses ];
```

Comment:
Does the program link against ncurses? If it is only used in postPatch then this line does nothing

Reply:
You're right, it builds without it. Should this line be removed and ncurses only be referenced in the patch? Or should something be put in its place?

Reply:
The line can then be removed because it does nothing